src/platformdirs/android.py

```diff
@@ -56,6 +56,25 @@ def user_log_dir(self) -> str:
             path = os.path.join(path, "log")
         return path
 
+    @property
+    def user_documents_dir(self) -> str:
+        """
+        :return: documents directory tied to the user e.g. ``/data/media/0/Documents``
```

Comment:
This path not accessible on non-rooted devices, so it must be changed to `/storage/emulated/0/Documents`